libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -727,10 +727,10 @@ MAV_MISSION_RESULT AP_Mission::mavlink_to_mission_cmd(const mavlink_mission_item
 
         // sanity check location
         if (copy_location) {
-            if (fabsf(packet.x) > 90.0f) {
+            if (fabsf(packet.x) > 900000000) {
```

Comment:
This should be just abs() shouldn't it? packet.x is just a int32_t on mavlink_mission_item_int.